File: 4168-mirror-distance-of-an-integer/mirror-distance-of-an-integer.cpp
class Solution {
public:
    int rev(int n) {
        int sum = 0;
        while (n > 0) {
            sum = (sum * 10) + (n % 10);
            n /= 10;
        }
        return sum;
    }
    int mirrorDistance(int n) { return abs(n - rev(n)); }
};

File: 4168-mirror-distance-of-an-integer/mirror-distance-of-an-integer.java
class Solution {
    public int mirrorDistance(int n) {
        return (int) Math.abs(n - rev(n));
    }

    private int rev(int n) {
        long sum = 0;
        while (n > 0) {
            sum = sum * 10 + n % 10;
            n /= 10;
        }
        return (int) sum;
    }
}